lib/index.js

```diff
@@ -10,10 +10,13 @@ var oauth2_client;
  *  @returns {Object} oauth2_client - the Google OAuth2 client
  */
 function create_oauth2_client () {
+  var google_client_id = (OPTIONS.google_client_id) ? OPTIONS.google_client_id : process.env.GOOGLE_CLIENT_ID;
+  var google_client_secret = (OPTIONS.google_client_secret) ? OPTIONS.google_client_secret : process.env.GOOGLE_CLIENT_SECRET;
+  var base_url = (OPTIONS.base_url) ? OPTIONS.base_url : process.env.BASE_URL;
```

Comment:
only thing to _note_ here is that allowing the `GOOGLE_CLIENT_ID` and `GOOGLE_CLIENT_SECRET` to be passed in on the `OPTIONS` object has not been reflected in the Docs (README.md) ...
We _do_ want this feature as it makes the plugin more flexible. but we need to document it so other people are aware of it. 👍